app/views/merchant/dashboard/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<section id="merchant_details">
```

Comment:
I think a heading, such as 'Employer', 'Merchant' or similar, would be helpful to specify for these details.

Reply:
@freeheeling when you say, 'heading' do you mean changing the id #merchant_details to #employer'?